include/all.hpp

```diff
@@ -1,9 +1,11 @@
 #ifndef _all_hpp_
 #define _all_hpp_
 
+#include <Eigen/SVD>
 #include <Matrix.hpp>
 #include <assert.h>
 #include <iostream>
+#include <math.h>
```

Comment:
`math.h` is a C header library, it should be replaced by it's C++ counterpart `cmath` and likewise for `assert.h` and `stdlib.h`.